README.md

```diff
@@ -191,8 +193,12 @@ parameters:
         ticket:
             enabled: true
 
+            # one of: jira, github (case-sensitive)
+            tracker: jira
+
             # a case-sensitive list of status names.
             # only tickets having any of these statuses are considered resolved.
+            # supported trackers: jira. Other trackers ignore this parameter.
```

Comment:
@EmilMassey why does the github tracker not need the resolved statuses?

don't we need the github ticket status in this list here to overcome the IF in 
https://github.com/staabm/phpstan-todo-by/blob/00690bbd6d223078767fced0d099ca7a2ee86a46/src/TodoByTicketRule.php#L63-L65

?

Reply:
@staabm 
In GitHub there are only two issue states: `open` and `closed`. This is different than Jira, where you can define your statuses according to your needs. So setting this parameter doesn't make sense for GitHub, because you always want to check if it is `closed`. 

This condition you mentioned is satisfied because in `TicketRuleConfigurationFactory` we hardcode `['closed']` list as resolved statuses.